lib/cocoapods-core/source/metadata.rb

```diff
@@ -53,7 +53,7 @@ def path_fragment(pod_name, version = nil)
       def last_compatible_version(target_version)
         return unless minimum_cocoapods_version
         return if minimum_cocoapods_version <= target_version
-        @last_compatible_versions.reverse_each.bsearch { |v| v <= target_version }.tap do |version|
+        Array(@last_compatible_versions.reverse_each).bsearch { |v| v <= target_version }.tap do |version|
```

Comment:
this could be simplified as `@last_compatible_versions.reverse.bsearch`

Reply:
oo that's much better. will do